app/services/publisher/pingdom_eligibility.rb

```diff
@@ -0,0 +1,36 @@
+class Publisher
+  class PingdomEligibility
```

Comment:
### class Uptime::Eligibility

just coz we don't necessarily have to make it specific to pingdom

Reply:
great class name by the way 😆